source/js/image-request-handler.js

```diff
@@ -19,7 +19,7 @@ export default class ImageRequestHandler
         this.timeout = setTimeout(() => {
             // Initiate the request
             this._image = new Image();
-            this._image.crossOrigin = "anonymous";
+            //this._image.crossOrigin = "anonymous"; TODO: make configurable
```

Comment:
Could you please make this configurable in this PR? Add `settings.crossOrigin: "anonymous"` in the default settings, and then add the reference to the setting here.

Reply:
I didn't make this configurable here because `ImageRequestHandler` doesn't have access to `settings`. 

I need to put settings into the `options` that get passed to `ImageRequestHandler` upstream. I'll check that possibility.